buddyboss/buddyboss-platform
# Medium Editor > Image dragging not working

### All Submissions:






### Changes proposed in this Pull Request:

Fixes #840 

### How to test the changes in this Pull Request:

1. Go to Forums > Topic > Click on Reply
2. Drag and drop image in the form editor
3. It should not allow image dragging in the editor. Only through media icon.

### Other information:
